shiksha-migration/cohort-academicYear-userdb-migration.js

```
@@ -0,0 +1,185 @@
+async function migrateCohortAcademicYear() {
+  console.log('=== STARTING COHORT ACADEMIC YEAR MIGRATION ===');
+
+  const Client = new Client(dbConfig.source);
+
+  try {
+ 
+    await Client.connect();
+
+    console.log(
+      '[COHORT_ACADEMIC_YEAR] Connected to destination database'
+    );
```

ESLint

[error] Replace `⏎······'[COHORT_ACADEMIC_YEAR]·Connected·to·destination·database'⏎····` with `'[COHORT_ACADEMIC_YEAR]·Connected·to·destination·database'`

(prettier/prettier)